### DISCUSSION TITLE
Is it possible to use analogWrite() function together with running dscKeybusInterface ?

### DISCUSSION THREAD

**Discussion by UltimatPronin** [taligentx/dscKeybusInterface#170] [dscKeybusInterface Discussion]
I tried to use analogWrite() to do different blinks with different brightness for Nodemcu's  board build in leds to visual indicate module state (for example, Idle, Processing received command, busy, etc...) but get a lot of reboots. I think it is because timer1 is used by dscKeybusInterface. Maybe anyone can advice the proper way how to use timer1 together with analogWrite()?

**Comment by taligentx** [comment]
(Moved over to discussions)

> Maybe anyone can advice the proper way how to use timer1 together with analogWrite()?

I'm not sure if there's a good way to use `analogWrite()` - the esp8266 has two timers, and timer0 apparently is used for WiFi so timer1 is the remaining timer and critical for the library to function.  It would be interesting to see if there's any way to use timer0.

**Comment by UltimatPronin** [comment]
Looks like timer0 usage not possible if WiFi is needed.
OK, for now I use digitalWrite() without an issue.
